src/Components/Event/Event.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect} from 'react'
+import EventForm from '../EventForm/EventForm'
+//
+const Event = () => {
+  const [lifeEvent, setLifeEvent] = useState('')
+  const [startEvent, setStartEvent] = useState('')
+  const [endEvent, setEndEvent] = useState('')
+  const [display, changeDisplay] = useState(false)
+
+  const handleLifeEvent = (event:React.ChangeEvent<HTMLInputElement>) => {
+    setLifeEvent(event.target.value)
+  }
+
+  const handleStartEvent = (event:React.ChangeEvent<HTMLInputElement>) => {
+    setStartEvent(event.target.value)
+  }
+
+  const handleEndEvent = (event:React.ChangeEvent<HTMLInputElement>) => {
+    setEndEvent(event.target.value)
+  }
+
+  const handleClearEvent = (event:React.ChangeEvent<HTMLInputElement>) => {
+    setLifeEvent('')
+  }
+
+  const handleButtonClick = () => {
+    changeDisplay(true)
+  }
+
+  return (
```

Comment:
We'll have to get this mutation hooked up & then we're all set. Awesome job here!